metaflow/metaflow_config.py

```diff
@@ -81,6 +81,9 @@
 # so setting it to 0 means each operation will be tried once.
 S3_RETRY_COUNT = from_conf("S3_RETRY_COUNT", 7)
 
+# S3 worker configuration. Number of concurrent S3 connections.
```

Comment:
nit: "Number of concurrent S3 processes for parallel operations"

Reply:
done